src/qt/peertablemodel.cpp

```diff
@@ -10,7 +10,7 @@
 
 #include <interfaces/node.h>
 
-#include <algorithm>
+#include <utility>
```

Comment:
why the switch from `algorithm` to `utility`?

Reply:
Please look #17634.